sliver/hostuuid/uuid_windows.go

```diff
@@ -0,0 +1,38 @@
+// +build windows
+
+package hostuuid
+
+/*
+  Sliver Implant Framework
+  Copyright (C) 2019  Bishop Fox
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+import (
+	"bytes"
+	"os/exec"
+)
+
+func GetUUID() string {
+	cmd := exec.Command("wmic.exe", "csproduct", "get", "UUID")
```

Comment:
I'd like to avoid creating a sub process just for that, especially since calling `wmi.exe` is often flagged as suspicious.
I just did a quick test, and turns out you can get the UUID from the registry in `HKEY_LOCAL_MACHINE\SYSTEM\HardwareConfig\LastConfig`.
So I'd prefer to have you query that than resorting to calling a sub process.

Reply:
How about native wmi?
https://stackoverflow.com/questions/20365286/query-wmi-from-go

Reply:
I thought about it but it's adding a new 3rd party dependency, which we usually try to avoid due to the obfuscation process. The `golang.org/x/sys/windows` dependency is already present and allows you to query the registry quite easily.

Reply:
> Sounds good.
> 
> Because I am no longer depending on the x/sys/unix package, should I revert my changes in the string-obfuscator?
> Notably I changed the dynamic length calculation to static in order to prevent a syntax error.

Yes absolutely. I saw the change and wasn't sure what it was about. Please avoid modifying the obfuscator code as much as possible, unless your PR is about obfuscation. I first thought your commit was reverting a previous modification you made, so I have no commented on it.
However, if you spot issues in the obfuscator code, we'll be glad to accept fixes :)

Reply:
I'll add the obfuscator change in a separate pr.